code/modules/clothing/spacesuits/rig/rig.dm

```diff
@@ -455,6 +472,11 @@
 			for(var/obj/item/rig_module/module in installed_modules)
 				module.deactivate()
 			offline = 2
+			//Commented out until boots/helmets stay disabled. Spacesuits don't have a power restriction on them.
```

Comment:
As the rigsuits are still somewhat WIP, i did leave that codepart in untill a way is found to have a consistent handling, and not just the boots/light disables, but can be reactivated again. Don't mind if this goes here, however.